godel_surface_detection/package.xml

```diff
@@ -28,6 +28,7 @@
   <depend>godel_process_path_generation</depend>
   <depend>godel_param_helpers</depend>
   <depend>godel_plugins</depend>
+  <depend>godel_utils</depend>
```

Comment:
Another package.  I worry that there are already a lot of packages which makes it difficult for other developers to contribute.  

I think a utils meta-package may be appropriate.  Can we add the godel_param_helpers and godel_polygon_offset to this package?

I don't think this is an issue for this PR, but it is a good place to start thinking about it.